lib/r10k/cli/deploy.rb

```diff
@@ -23,6 +23,8 @@ def self.command
 
         required :c, :config, 'Specify a configuration file'
 
+        required nil, :cachedir, 'Specify a cachedir, overriding the value in config'
```

Comment:
If we're focusing on getting the required functionality now and expanding things later, let's not worry about `r10k deploy module` right now.